woocommerce/class-sv-wc-plugin.php

```diff
@@ -706,6 +706,7 @@ public function add_system_status_php_information( $rows ) {
 					continue;
 				}
 
+				/* translators: Placeholders: %1$s - PHP setting value, %2$s - version or value currently set */
```

Comment:
The message itself suggests a different placeholder context, no?

```suggestion
				/* translators: Placeholders: %1$s - PHP setting value, %2$s - version or value required */
```